package.json

```diff
@@ -6,7 +6,12 @@
   "directories": {
     "test": "test"
   },
+  "watch": {
```

Comment:
Can we do this the same as the client-js?

"watch": "node node_modules/watch/cli.js \"npm test\" ./src ./test-unit ./test-e2e"